docs/usage.rst

```diff
@@ -92,9 +92,74 @@ all
 Unittest Integration
 --------------------
 
-While it's possible to use the context manager or decorator forms with unittest,
-there's also a ``VCRTestCase`` provided separately by `vcrpy-unittest
-<https://github.com/agriffis/vcrpy-unittest>`__.
+Inherit from ``VCRTestCase`` for automatic recording and playback of HTTP
+interactions.
+
+.. code:: python
+
+    from vcr.unittest import VCRTestCase
+    import requests
+
+    class MyTestCase(VCRTestCase):
+       def test_something(self):
+           response = requests.get('http://example.com')
+
+Similar to how VCR.py returns the cassette from the context manager,
+``VCRTestCase`` makes the cassette available as ``self.cassette``:
+
+.. code:: python
+
+    self.assertEqual(len(self.cassette), 1)
+    self.assertEqual(self.cassette.requests[0].uri, 'http://example.com')
+
+By default cassettes will be placed in the ``cassettes`` subdirectory next to the
+test, named according to the test class and method. For example, the above test
+would read from and write to ``cassettes/MyTestCase.test_something.yaml``
+
+The configuration can be modified by overriding methods on your subclass:
+``_get_vcr_kwargs``, ``_get_cassette_library_dir`` and ``_get_cassette_name``.
+To modify the ``VCR`` object after instantiation, for example to add a matcher,
+you can hook on ``_get_vcr``, for example:
+
+.. code:: python
+
+    class MyTestCase(VCRTestCase):
+        def _get_vcr(self, **kwargs):
+            myvcr = super(MyTestCase, self)._get_vcr(**kwargs)
+            myvcr.register_matcher('mymatcher', mymatcher)
+            myvcr.match_on = ['mymatcher']
+            return myvcr
+
+See
+`the source
+<https://github.com/kevin1024/vcrpy/blob/master/vcr/unittest/testcase.py>`__
+for the default implementations of these methods, and `VCR.py`_ for more
```

Comment:
probably don't need to refer users to VCR.py here, now that the repos are combined

Reply:
Good point, dropped, thanks! :+1: